enhancements/cluster-logging/forward_to_cloudwatch.md

```diff
@@ -0,0 +1,198 @@
+---
+title: forward_to_cloudwatch
+authors:
+  - "@alanconway"
+reviewers:
+  - "@jcantrill"
+  - "@jeremyeder"
+approvers:
+creation-date: 2020-12-17
+last-updated: 2020-12-17
+status: implementable
+see-also:
+superseded-by:
+---
+
+# Forward to Cloudwatch
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+[Amazon Cloudwatch][aws-cw] is a hosted monitoring and log storage service.
+This proposal extends the `ClusterLogForwarder` API with an output type for Cloudwatch.
+
+## Motivation
+
+Amazon Cloudwatch is a popular log store.
+We have requests from external and  Red Hat-internal customers to support it.
+
+### Goals
+
+Enable log forwarding to Cloudwatch.
+
+### Non-Goals
+
+Enable Cloudwatch metric collection.
+
+## Proposal
+
+New API fields in the `output.cloudwatch` section:
+
+- `region`: (string) AWS region name.
```

Comment:
What will be done with the `region` parameter?  Will it be used to construct the the proper API endpoint from [1]?

[1] https://docs.aws.amazon.com/general/latest/gr/cwl_region.html

Reply:
Guess this would be useful if the CW endpoint configured (via the credentials I assume) is not in the same region as the cluster.  Is this optional and defaults to cluster's region if not ste?

Reply:
I believe @portante originally mentioned to me this would be useful.  I don't pretend to understand what the implications of specifying it.  The default would be to do nothing with region if it is not set

Reply:
As I read it, region is required to establish a cloudwatch connection. Its my first reading though, so if that's not correct we'll adjust. Does anyone have a working cloudwatch configuration that we can compare with this proposal?